src/shared/reducers/autoRefresh.ts

```diff
@@ -38,7 +38,7 @@ export const autoRefreshReducer = (state = initialState(), action: Action) =>
           draftState[dashboardID] = AUTOREFRESH_DEFAULT
         }
 
-        draftState[dashboardID].status = status
+        draftState[dashboardID] = {...draftState[dashboardID], status}
```

Comment:
it's weird that this would resolve the issue. I think that's a red-herring since the draftState shouldn't need to set the object in its entirety. Could it be that `status` just doesn't exist on the initial object and that's the issue here?